analyzers/src/SonarAnalyzer.Common/Helpers/KnownAssembly.cs

```diff
@@ -39,16 +39,26 @@ public sealed partial class KnownAssembly
         new(And(NameIs("Microsoft.VisualStudio.QualityTools.UnitTestFramework").Or(NameIs("Microsoft.VisualStudio.TestPlatform.TestFramework")),
                 PublicKeyTokenIs("b03f5f7f11d50a3a")));
 
-    public static KnownAssembly FluentAssertions { get; } = new(NameIs("FluentAssertions").And(PublicKeyTokenIs("33f2691a05b67b6a")));
```

Comment:
Suggestion to reviewer: Read this file starting from the bottom, I added a helper constructor to avoid duplication.